build/scripts/bootstrap.sh

```diff
@@ -68,21 +68,28 @@ function installBuildDependencies
 
         apt-get install -y make build-essential g++ gdb gdbserver gcc git wget
         apt-get install -y python3 ninja-build
-        apt-get install -y cmake libmsgsl-dev
         apt-get install -y libboost-program-options-dev
         apt-get install -y libproxy-dev libssl-dev uuid-dev libcurl4-openssl-dev
 
+        if [[ $DISTRO == "ubuntu" && $VER == "22.04" ]];
+            then apt-get install -y cmake libmsgsl-dev=4.0.0-2;
```

Comment:
>then [](http://example.com/codeflow?start=12&length=4)

'then' should be indented one level left and the apt-get cmdline should be on its own line.